Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/PDFDiscretePresentationController.mm

```diff
@@ -1095,24 +1095,25 @@ static float elasticDeltaForTimeDelta(float initialPosition, float initialVeloci
         createRowPageBackgroundContainerLayers(rowIndex, layoutRow, row);
 
         // This contents layer is used to paint both pages in two-up; it spans across both backgrounds.
-        row.contentsLayer = createGraphicsLayer(makeString("Row contents "_s, rowIndex), GraphicsLayer::Type::TiledBacking);
-        row.contentsLayer->setAnchorPoint({ });
-        row.contentsLayer->setDrawsContent(true);
-        row.contentsLayer->setAcceleratesDrawing(m_plugin->canPaintSelectionIntoOwnedLayer());
+        RefPtr rowContentsLayer = row.contentsLayer;
```

Comment:
What is this assignment for? You're reassigning the variable unconditionally on the next line..